dawg_python/wrapper.py

```diff
@@ -17,29 +17,29 @@ def __init__(self):
     "Root index"
 
     def has_value(self, index):
-        "Checks if a given index is related to the end of a key."
+        #Checks if a given index is related to the end of a key.
```

Comment:
I think using triple quotes and replacing "checks/gets/reads/..." with "check/get/read" should make them pep8-compatible.